radis/io/query.py

```diff
@@ -79,6 +77,9 @@ def fetch_astroquery(
     :py:attr:`astroquery.query.BaseQuery.cache_location`
 
     """
+    from astropy import units as u
```

Comment:
Basically import packages when necessary (?)

Reply:
Yes ! Imports within functions is harder to maintain so you don't want to do that for things you'll have to load anyway. 

For astropy.units it didn't really work; as it's still imported at one point to check the inputs.